cumulus/parachains/integration-tests/emulated/common/src/macros.rs

```diff
@@ -130,3 +130,279 @@ macro_rules! test_parachain_is_trusted_teleporter {
 		}
 	};
 }
+
+#[macro_export]
```

Comment:
We need to publish a new version of `emulated-integration-tests-common` to be able to use this in the fellowship/runtimes repo.

@programskillforverification please also add a [prdoc](https://github.com/paritytech/polkadot-sdk/tree/master/prdoc) with short changelog description and include at least `emulated-integration-tests-common` (bump minor) there.

Reply:
> We need to publish a new version of `emulated-integration-tests-common` to be able to use this in the fellowship/runtimes repo.
> 
> @programskillforverification please also add a [prdoc](https://github.com/paritytech/polkadot-sdk/tree/master/prdoc) with short changelog description and include at least `emulated-integration-tests-common` (bump minor) there.

I will add it asap